text/survey_analysis_report2.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "A folder structure to organise files in a reserch project"
+bibliography:  "GIN-tonic-paper-zotero.bib"
+author:
+  - Julien Colomb
+  - The Gin-tonic team
+output: 
+  bookdown::html_document2:
+    keep_md: true
+
+---
+
+
+![titleimage: Some positive feedback received in the survey.](../50_figures/pulication_ready_figures/blogtitle.jpeg)
+
+# Summary
+
+In contrast to software developers, researchers have no standard way to organise their files.
+While data managers recommend to design such an organisation,
+they fail at providing clear recommendations or examples;
+and there is no time allocated to this task in the researcher work.
+We believe that providing researcher with a folder tree structure template could make a huge difference in promoting data management and research collaboration.
+Here we present the results of an initial survey run in three neuroscience research consortium (LINK).
+
+
+We collected the feedback from 51 neuroscientists presented with 2 initial drafts,
+analysed their responses,
+and build a second template draft based on this feedback that is presented here.
+This new template takes three levels of data organisation into account (experiment, project, laboratory),
+while staying relatively simple and flexible, especially when implemented it in a git workflow (using the GIN-tonic tool we are developing).
```

Comment:
while staying fairly simple and flexible, especially when implemented it in a git workflow (using the GIN-tonic tool we are developing).